pyproject.toml

```diff
@@ -18,7 +18,8 @@ dependencies = [
         "chaco==5.1.0",
         "click==8.1.3",
         "enable==5.3.1",
-        "numpy==1.23.3",
+        # "numpy==1.23.3",
+        "intel-numpy==1.21.4",
```

Comment:
If the user's processor is not intel, does this fall back to traditional numpy?  I would also suggest documenting this requirement in the README.

Reply:
Great question! I don't know the answer. :(

I like the idea of adding something about this to the README, but am not sure what is appropriate to say.
Thoughts?

Reply:
I guess we just link to the wiki for installing.  I'll take an action to add a pip section to https://github.com/capn-freako/PyBERT/wiki/instant_gratification

Reply:
> I guess we just link to the wiki for installing. I'll take an action to add a pip section to https://github.com/capn-freako/PyBERT/wiki/instant_gratification

Thanks! :)

Reply:
Great info on _NumPy_ w/ new Apple silicon here:
https://stackoverflow.com/questions/70240506/why-python-native-on-m1-max-is-greatly-slower-than-python-on-old-intel-i5